make/autoconf/jdk-options.m4

```diff
@@ -665,7 +665,7 @@ AC_DEFUN([JDKOPT_ENABLE_DISABLE_COMPATIBLE_CDS_ALIGNMENT],
   UTIL_ARG_ENABLE(NAME: compatible-cds-alignment, DEFAULT: $COMPATIBLE_CDS_ALIGNMENT_DEFAULT,
       RESULT: ENABLE_COMPATIBLE_CDS_ALIGNMENT,
       DESC: [enable use alternative compatible cds core region alignment],
-      DEFAULT_DESC: [disabled],
+      DEFAULT_DESC: [disabled except on linux-aarch64],
```

Comment:
Accepting this updated default from upstream would be misleading.
Also, line 199 should be deleted like it was upstream.

Reply:
Restored `DEFAULT_DESC: [disabled]`, and removed `AC_SUBST(COMPATIBLE_CDS_ALIGNMENT_DEFAULT)`.